.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,7 +1,6 @@
 ---
 name: Request a feature
-about: Use this template to request new functionality or change behavior of the
-library
+about: Use this template to request new functionality or change behavior of the library
```

Comment:
"change **the** behavior of the library"?

Reply:
fixed